conf/kpl.properties

```diff
@@ -113,7 +113,7 @@ LogLevel = info
 # Default: 4
 # Minimum: 1
 # Maximum (inclusive): 128
-MaxConnections = 16
+MaxConnections = 4
```

Comment:
It was part of an effort to revert all KPL settings back to default. I changed this because the description for this config mentioned: 

```
# Setting this too high may impact latency and consume additional resources
# without increasing throughput.	
```

I haven't tested the performance with `MaxConnections=16`, something I can play with.

Reply:
Apparently `MaxConnections` doesn't affect performance in my tests, so I'm reverting back to `16`.